app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
-  devise :database_authenticatable,
-    :registerable, :jwt_authenticatable,
+  devise :database_authenticatable, :recoverable, 
+    :registerable, :jwt_authenticatable, 
```

Comment:
this space is not necessary 💅🏻 
```suggestion
    :registerable, :jwt_authenticatable,
```